packages/design-system-tokens/src/lib/utility.ts

```diff
@@ -1,13 +1,127 @@
 import fs from 'fs';
 import path from 'path';
-import { HexValue, RGBValue, FileDescriptor } from './types';
+import { HexValue, RGBValue, FileDescriptor, PercentageValue } from './types';
 
 // converts an rgb string 'rgb(15,24,128)' to a hex value '#0819A9'
 export const rgbToHex = (r: number, g: number, b: number): HexValue => {
   const hex = ((1 << 24) + (r << 16) + (g << 8) + b).toString(16).slice(1);
   return `#${hex}`;
 };
 
+/* Table of Alpha Values for 10 bit Hex
+ *
+ * Alpha% 	Hex	Num
+ * ----------------
+ * 100%	    FF	255
```

Comment:
If we were supplying this value as hex, this table would be useful, but aren't we only ever supplying the alpha value as a percentage?